Makefile

```diff
@@ -244,7 +248,7 @@ refresh-notebooks:
 NB_REFRESH_FILE := ""
 .PHONY: refresh-notebooks-impl
 refresh-notebooks-impl:
-	bash -c 'WPS_URL="$(WPS_URL)" FINCH_WPS_URL="$(FINCH_WPS_URL)" FLYINGPIGEON_WPS_URL="$(FLYINGPIGEON_WPS_URL)" jupyter nbconvert --to notebook --execute --ExecutePreprocessor.timeout=240 --output "$(NB_REFRESH_FILE)" "$(NB_REFRESH_FILE)"; sed -i "s@$(WPS_URL)/outputs/@$(OUTPUT_URL)/@g" "$(NB_REFRESH_FILE)"'
+	bash -c 'WPS_URL="$(WPS_URL)" FINCH_WPS_URL="$(FINCH_WPS_URL)" FLYINGPIGEON_WPS_URL="$(FLYINGPIGEON_WPS_URL)" jupyter nbconvert --to notebook --execute --ExecutePreprocessor.timeout=240 --output "$(NB_REFRESH_FILE)" --output-dir . "$(NB_REFRESH_FILE)"; sed -i "s@$(WPS_OUTPUT_URL)/@$(OUTPUT_URL)/@g" "$(NB_REFRESH_FILE)"'
```

Comment:
Won't changing the output dir to `.` place the notebook at the location where this command is called? Why not use `in-place`? See: https://nbconvert.readthedocs.io/en/latest/usage.html#notebook-and-preprocessors

Reply:
Oh funny, I didn't notice that `--inplace` option yesterday with `jupyter nbconvert --help`.  However setting `--output "$(NB_REFRESH_FILE)" --output-dir .` does exactly the same thing since `$(NB_REFRESH_FILE)` is the full path of the notebook.